examples/lock-app/efr32/BUILD.gn

```diff
@@ -184,7 +184,10 @@ efr32_executable("lock_app") {
 
   if (chip_enable_pw_rpc || chip_build_libshell || enable_openthread_cli ||
       use_wf200 || use_rs911x) {
-    sources += [ "${examples_plat_dir}/uart.cpp" ]
+    sources += [
+      "${examples_plat_dir}/uart.cpp",
+      "src/EventHandler.cpp",
```

Comment:
I believe this new file should only be added for `chip_build_libshell`

Reply:
moved to its own section